wiki/Beatmaps/en.md

```diff
@@ -1,3 +1,28 @@
+### Beatmaps
+
+A beatmap (also known as a _level_ or a _map_) is where the majority of _osu!_ lies.
+These could be related to those in the stages in the DS games.
+In their most basic form, beatmaps have a song and hit objects placed based on the rhythm of the music.
+
+Nowadays, beatmaps usually have a background image, video, or even a storyboard.
+
+For more details about how beatmaps are uploaded, see BSS.
+
+### Beatmap Pack
+
+_Not to be confused with Beatmapsets_
```

Comment:
the common convention for disambiguation is to use
```
_Not to be confused with {item}._
```
or at least, that is what Wikipedia uses...